core/controllers/ingest.py

```diff
@@ -31,13 +32,9 @@ def ingest(body):
     if validation_failures:
         raise ValidationError(validation_failures=validation_failures)
 
-    data = {}
-    for sheet_name, sheet in workbook.items():
-        sheet_as_dict = sheet.fillna("").to_dict(orient="split")
-        del sheet_as_dict["index"]
-        data[sheet_name] = sheet_as_dict
+    current_app.db = FakeDB.from_dataframe(workbook)
 
-    return data, 200
+    return "Success: Data ingested", 200
```

Comment:
Small point but we could align this with the example response in `openapi/components.yml`

Reply:
Sure thing